sdk/eventhub/event-hubs/tsconfig.json

```diff
@@ -5,6 +5,6 @@
     "outDir": "./dist-esm",
     "downlevelIteration": true
   },
-  "exclude": ["node_modules", "./types/**/*.d.ts", "./samples/**/*.ts"],
+  "exclude": ["node_modules", "./types/**/*.d.ts", "./samples/**/*.ts", "test/perf/track-1/"],
```

Comment:
Do we need to include track-2 in here as well?

Reply:
No, we want it to be compiled with the regular tests.